### PR TITLE
XML code block designation on page is causing issues with display/ren…

### PR DESCRIPTION
**Pull request by markshust**
XML code block designation on page is causing issues with display/rendering on frontend

## Purpose of this pull request

The display of the documention on the frontend of this site is not rendering correcting.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/cloud/trouble/trouble_ce-creds.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by hguthrie** [comment]
Whoa! That's freaky.

File: guides/v2.2/cloud/trouble/trouble_ce-creds.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ To resolve the issue with credentials:
 
    {% collapsible Click to show/hide snippet %}
 
-   ```xml
+   ```
```
Comment (hguthrie) [inline guides/v2.2/cloud/trouble/trouble_ce-creds.md]:
```suggestion
   ```terminal
```

File: guides/v2.2/cloud/trouble/trouble_ce-creds.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ To resolve the issue with credentials:
 
    {% collapsible Click to show/hide snippet %}
```
Comment (hguthrie) [inline guides/v2.2/cloud/trouble/trouble_ce-creds.md]: Remove this line and the `endcollapsible` below...

Comment (markshust) [reply guides/v2.2/cloud/trouble/trouble_ce-creds.md]: thanks, all removed. i'd look into why this is happening with the collapsible tag but i don't think it's really even needed here anyways. should be all set now. thanks for the `terminal` recommendation. how are you viewing/previewing the markdown files locally to confirm they look ok? i'm using macdown but apparently it's not a 1:1 preview with how github displays markdown.

Comment (hguthrie) [reply guides/v2.2/cloud/trouble/trouble_ce-creds.md]: I am building the site locally using a custom `_config.local.yaml` to exclude all guides except for Cloud. Makes it faster. 😉

Comment (hguthrie) [reply guides/v2.2/cloud/trouble/trouble_ce-creds.md]: https://github.com/magento/devdocs#building-this-site

**Pull request merged**